pwiz_tools/Skyline/Util/CreateHandleDebugBase.cs

```diff
@@ -263,13 +263,5 @@ private void FormCreateHandle()
                 }
             }
         }
-

```

Comment:
This seems like an unrelated change.

Reply:
I'll put this back in and remove it in a different pull request.